libcudacxx/test/libcudacxx/cuda/for_each_canceled/for_each_canceled.pass.cpp

```diff
@@ -0,0 +1,220 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: pre-sm-100
+// UNSUPPORTED: !nvcc
+// UNSUPPORTED: nvrtc
+// UNSUPPORTED: c++98, c++03, c++14, c++17
```

Comment:
Please make it possible on C++17 as well. It shouldn't be hard. At least in a follow-up PR.
```suggestion
// UNSUPPORTED: c++98, c++03, c++14
```

Reply:
I think I just forgot to enable the tests here, thanks!